modules/pairIdSystem.js

```diff
@@ -90,19 +95,20 @@ export const pairIdSubmodule = {
           logInfo('Pairid: Error parsing JSON: ', error);
         }
       } else {
-        logInfo('Pairid: liverampValue for pairId from storage is empty or null');
+        logInfo('Pairid: data clean room value for pairId from storage is empty or null');
       }
     }
 
     if (ids.length == 0) {
       logInfo('PairId not found.')
       return undefined;
     }
+
     return {'id': ids};
   },
   eids: {
     'pairId': {
-      source: 'google.com',
+      source: 'iabtechlab.com',
```

Comment:
In future IAB may support another Alt ID, so the source for PAIR should clearly mention what it is. 
Can we mention it as "pair-iab-tl.com"?

Reply:
isn't that what atype is for? like a single source can have multiple id types?

Reply:
Here are the existing `atype` definitions https://github.com/InteractiveAdvertisingBureau/AdCOM/blob/main/AdCOM%20v1.0%20FINAL.md#list_agenttypes

Reply:
atype identifies the user agent types a user identifier is from.

Reply:
@therevoltingx mentioned we will have a unique atype to identify eid as a PAIR